phoenix-spark/src/main/scala/org/apache/phoenix/spark/PhoenixRelation.scala

```diff
@@ -42,8 +42,8 @@ case class PhoenixRelation(tableName: String, zkUrl: String, dateAsTimestamp: Bo
       requiredColumns,
       Some(buildFilter(filters)),
       Some(zkUrl),
-      new Configuration(),
-      dateAsTimestamp
+      conf = new Configuration(),
```

Comment:
I could have placed tenantId at the end of the PhoenixRDD class so we don't have to used the named parameters "conf" and "dateAsTimeStamp". I'll commit a change for that.